src/Codeception/Test/Metadata.php

```diff
@@ -98,123 +104,83 @@ public function getCurrent($key = null)
         return $this->current;
     }
 
-    public function setCurrent(array $currents)
+    public function setCurrent(array $currents): void
     {
         $this->current = array_merge($this->current, $currents);
     }
 
-    /**
-     * @return mixed
-     */
-    public function getName()
+    public function getName(): string
     {
         return $this->name;
     }
 
-    /**
-     * @param mixed $name
-     */
-    public function setName($name)
+    public function setName(string $name): void
     {
         $this->name = $name;
     }
 
-    /**
-     * @return mixed
-     */
-    public function getFilename()
+    public function getFilename(): string
     {
         return $this->filename;
     }
 
-    /**
-     * @param mixed $index
-     */
-    public function setIndex($index)
+    public function setIndex($index): void
     {
         $this->index = $index;
     }
 
-    /**
-     * @return mixed
-     */
     public function getIndex()
     {
         return $this->index;
     }
 
-    /**
-     * @param mixed $filename
-     */
-    public function setFilename($filename)
+    public function setFilename(string $filename): void
     {
         $this->filename = $filename;
     }
 
-    /**
-     * @return array
-     */
-    public function getDependencies()
+    public function getDependencies(): array
     {
         return $this->params['depends'];
     }
 
-    public function isBlocked()
+    public function isBlocked(): bool
     {
         return $this->getSkip() !== null || $this->getIncomplete() !== null;
     }
 
-    /**
-     * @return mixed
-     */
-    public function getFeature()
+    public function getFeature(): ?string
```

Comment:
If this `getFeature` method can return `null`, then all other `getFeature` methods can return `null` too.